src/index.js

```diff
@@ -1,6 +1,9 @@
 let btn = document.querySelector(".press")
 btn.addEventListener("click", flip)
 
+let btn = document.querySelector(".press");
+btn.addEventListener("click", flip);
+
```

Comment:
This code already exists on the lines above, now it's there twice, meaning our button will have 2 eventListeners that flip the board and this could cause a bug! Please keep only one copy, thanks :)